models/recruit/src/index.ts

```diff
@@ -46,6 +46,9 @@ export class TCandidate extends TPerson implements Candidate {
   @Prop(TypeString(), 'Attachments' as IntlString)
   attachments?: number
 
+  @Prop(TypeString(), 'Comments' as IntlString)
+  comments?: number
```

Comment:
I've could not find there is it updated?

Reply:
server/core/storage:109 here